doc/sphinx/package_service_mapping.json

```diff
@@ -868,7 +896,121 @@
   "azure-mgmt-appconfiguration": {
     "category": "Management",
     "service_name": "App Configuration",
-    "manually_generated": true
+    "namespaces": [
+      "azure.mgmt.appconfiguration"
+    ]
+  },
+  "azure-mgmt-privatedns": {
+    "category": "Management",
+    "service_name": "Private DNS",
+    "namespaces": [
+      "azure.mgmt.privatedns"
+    ]
+  },
+  "azure-mgmt-frontdoor": {
+    "category": "Management",
+    "service_name": "Network",
+    "namespaces": [
+      "azure.mgmt.frontdoor"
+    ]
+  },
+  "azure-mgmt-hybridcompute": {
+    "category": "Management",
+    "service_name": "Hybrid Compute",
```

Comment:
Can't find a single service name -  I see some `Azure Hybrid` or `Azure Hybrid Cloud solutions`. I think there is no equivalent of this sdk in JS (can't find one in our monorepo)

Reply:
https://azure.microsoft.com/en-us/services